qml/Stage/Stage.qml

```diff
@@ -767,11 +768,15 @@ FocusScope {
                 }
                 z: normalZ
 
+                // Set these as propertyes as they wont update otherwise
+                property real screenOffsetX: Screen.virtualX
+                property real screenOffsetY: Screen.virtualY
+
                 // Normally we want x/y where the surface thinks it is. Width/height of our delegate will
                 // match what the actual surface size is.
                 // Don't write to those, they will be set by states
-                x: model.window.position.x - clientAreaItem.x
-                y: model.window.position.y - clientAreaItem.y
+                x: model.window.position.x - clientAreaItem.x - screenOffsetX
+                y: model.window.position.y - clientAreaItem.y - screenOffsetY
```

Comment:
Isn't this X/Y relative to the parent item i.e. the stage i.e. current screen?

Reply:
So unity8's and mir positions are different, unity8 uses the position of just the current screen (only once screen can have a shell currently)

unity8 x,y will be relative to the current screen only, so we need to remove the offset that comes from model.window.position as they will include the offset

An example, where we have two screens, screen 1 at 0,0 and screen 2 at 1200,0, and we place a application at the second screen at 1210,0

Unity8 will then place this as x,y 10,0 on screen 2, while mir will place this at 1210,0. So in simple term, unity8 x,y have no idea what screen its on as unity8's shell currently can only live on one screen anyway

This will be fixed by the really big pr: https://github.com/ubports/unity8/pull/164, but that wont be merged before OTA-12 has landed, so this is quick hack for the current version to have correct scaling in convergence mode.

Reply:
Just noticed, it's "-", not "+". Side effect of working too late. :D